sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1230,6 +1224,16 @@ object SQLConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(Long.MaxValue)
 
+  val ENABLE_FALL_BACK_TO_HDFS_FOR_STATS = buildConf("spark.sql.statistics.fallBackToHdfs")
+    .doc("This flag is effective only if it is Hive table. When true, it will fall back to HDFS " +
```

Comment:
`Hive table` -> `Non-partitioned Hive table`?

Reply:
Currently Hive partitioned table is supported, do you think we need to disable it?

https://github.com/apache/spark/blob/07c4b9bd1fb055f283af076b2a995db8f6efe7a5/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveStrategies.scala#L116-L136

Reply:
Yea I think so. Can you send a new PR to fix it?

Reply:
OK.